Content.Client/UserInterface/Systems/Chat/Widgets/ChatBox.xaml.cs

```diff
@@ -21,14 +22,23 @@ public partial class ChatBox : UIWidget
 {
     private readonly ChatUIController _controller;
     private readonly IEntityManager _entManager;
+    [Dependency] private readonly IConfigurationManager _cfg = default!; // EE - Chat stacking
+    [Dependency] private readonly ILocalizationManager _loc = default!; // EE - Chat stacking
 
     public bool Main { get; set; }
 
     public ChatSelectChannel SelectedChannel => ChatInput.ChannelSelector.SelectedChannel;
 
+    // EE - Chat stacking
+    private int _chatStackAmount = 0;
+    private bool _chatStackEnabled => _chatStackAmount > 0;
```

Comment:
Fix naming for get-only Property
```suggestion
    private bool ChatStackEnabled => _chatStackAmount > 0;
```